### ISSUE TITLE
SslConnectionConformanceTests:Flow_control failure

### ISSUE THREAD

**Issue by pepone** [icerpc/icerpc-csharp#3682]
```
2023-10-10T13:29:14.5541922Z   Failed IceRpc:Tests:Transports:Tcp:SslConnectionConformanceTests:Flow_control [7 s]
2023-10-10T13:29:14.5542460Z   Stack Trace:
```

This happen with .NET 8 RC1 on Ubuntu CI machine 

https://github.com/icerpc/icerpc-csharp/pull/3681/checks

**Comment by externl** [comment]
The URL is expired, not sure if there's a better one we can use.

**Comment by pepone** [comment]
Updated the URL to use the PR one, there isn't much info in the log anyway, the tests doesn't finish with the 8s timeout and gets kill by NUnit.

**Comment by pepone** [comment]
Fixed in 843b6b85306ffb70c1124c56e27e70802a810371 this was due to running many workers on Slow VMs
